docs/_guide/03-template-reference.md

```diff
@@ -309,9 +309,14 @@ const myTemplate = () => html`
 
 ### guard
 
-`guard(expression, valueFn)`
+`guard(expressions, valueFn)`
+
+Avoids re-evaluating an expensive template function (`valueFn`) unless one of the identified expressions changes identity. Returns the value of `valueFn`, which may be cached.
+
+The `expressions` argument can either be a single (non-array) expression, or an array of multiple expressions to monitor. 
+
+The `guard` directive caches the last-known value of `valueFn`, and only re-evaluates `valueFn` if the identity of any of the expressions changes (for example when a primitive changes value or when an object reference changes). 
```

Comment:
It doesn't actually cache the last result of `f`, it just doesn't update the DOM if it doesn't call `f`

Reply:
OK. But it's caching the generated DOM, then? Not sure whether it's returning nodes or a TemplateResult here...

Reply:
Oh, or... It's a directive, it just doesn't call setValue?

Reply:
Yes; if the `expressions` don't change, it just does nothing. Only if they change, it will evaluate the value function and update the DOM with the resulting values.